covalent/_shared_files/defaults.py

```diff
@@ -63,6 +63,7 @@ def get_default_sdk_config():
         ),
         "no_cluster": "true" if os.environ.get("COVALENT_DISABLE_DASK") == "1" else "false",
         "exhaustive_postprocess": "true",
+        "reconstruct_postprocess": "false",  # Note that exhaustive and eager options are mutually exclusive.
     }
```

Comment:
If they are mutually exclusive then don't we just need 1 variable indicating which one to use 🤔 .

Reply:
ah sorry they're not mutually exclusive as I realized recently - let me remove the comment

Reply:
Agreed, `exhaustive_postprocess` alone should be sufficient.